onmt/translate/translator.py

```diff
@@ -298,6 +304,34 @@ def _gold_score(
             gs = [0] * batch_size
         return gs
 
+    def translate_strings(self, input_sentences=[], transform=None):
+        """Translate a list of strings.
+        Args:
+            input_sentences: list of raw strings (not yet transformed sentences)
+        """
+        transforms_cls = get_transforms_cls(self.infer_opt._all_transform)
+        infer_iter = build_dynamic_dataset_iter(
+            self.infer_opt,
+            transforms_cls,
+            self.vocabs,
+            task=CorpusTask.INFER,
+            src=input_sentences,
+        )
+
+        infer_iter = IterOnDevice(infer_iter, self._gpu)
+        _, preds = self._translate(
+            infer_iter,
+            transform=infer_iter.transform,
+            attn_debug=self.infer_opt.attn_debug,
+            align_debug=self.infer_opt.align_debug,
+        )
+
+        output_sentences = [
```

Comment:
I don't think this part should be included in this method. it was specific to the MMLU evaluation

Reply:
I passed `attn_debug` and `align_debug` as arguments with the default value False.